### ISSUE TITLE
how do i add custom labels?

### ISSUE THREAD

**Issue by yosefy** [fstab/grok_exporter#8]
i want to add just x="y"
no connection to grok fields
may i?

thanks

**Comment by fstab** [comment]
Hi yosefy, it is currently not possible to do that in `grok_exporter`, but you can add labels in Prometheus using the `static_config` in the job configuration. For example, to add the `x=y` label for each metric from the `grok_exporter` instance on [localhost:9441](http://localhost:9441), put the following in `prometheus.yml`:

```yml
  - job_name: 'grok'
    static_configs:
      - targets: ['localhost:9144']
        labels: { 'x': 'y' }
```

If you want something less static (like dynamically rewriting labels depending on their values), check out Prometheus' `relabel_config`: https://prometheus.io/docs/operating/configuration/#relabel_config

Hope this helps
Fabian.

**Comment by fstab** [comment]
Hi, I just released `v0.2.0`, which uses templates for defining labels. With the new version, defining a constant label for grok_exporter should be easily possible like this:

```yaml
labels:
    x: 'y'
```

**Comment by yosefy** [comment]
cool!
thanks

**Comment by fstab** [comment]
Just released `v0.2.1` with even better support for templates. Closing this issue.
